meerkat_abacus/pipeline.py

```diff
@@ -81,15 +81,19 @@ def process_chunk(internal_buffer, session, engine, param_config=config,
             session,
             engine,
             **kwargs)
-        uuids += new_uuids
+        uuids.setdefault(form, [])
```

Comment:
uuids could be a default dict.
https://stackoverflow.com/a/3483652
```python
uuids = defaultdict(list)
```
Also the above post suggests a little more efficient and readable way:
```python
form_uuids = uuids.setdefault(form,[])
form_uuids.append(new_uuids)
```